.github/workflows/differential-shellcheck.yml

```diff
@@ -0,0 +1,17 @@
+name: Differential ShellCheck
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+
+    steps: 
+      - name: Repository checkout
+        uses: actions/checkout@v2
+
+      - name: Differential ShellCheck
+        uses: redhat-plumbers-in-action/differential-shellcheck@v1
```

Comment:
https://github.com/redhat-plumbers/dracut-rhel9/commit/32524aa0f72f9e63198927c22a9dcf37b497d538 is missing.

Reply:
Ah, I pushed to the wrong branch again... I hope it's the last time I use `master` instead of `main` for a PR....